Snowdog_SolrNavigation/templates/grid.phtml

```diff
@@ -106,40 +113,40 @@ $solrDataHelper = $this->helper('Snowdog\SolrNavigation\Helper\Data');
                                     </svg>
                                 </button>
                             </form>
-                            <form
-                                class="catalog-grid-item__secondary-action"
-                                action="<?= $block->escapeHtml($block->getAddToUrl('wishlist', $_product['entity_id'])) ?>"
-                                method="post"
-                                data-mage-init='{ "validation": {} }'
-                            >
-                                <button
-                                    class="button button--icon"
-                                    type="submit"
+                            <div class="catalog-grid-item__secondary-action">
+                                <a
+                                    class="button button--icon action towishlist"
```

Comment:
Do we need `action towishlist` classes?

Reply:
Same for `action tocompare` + in `list.phtml`

Reply:
Unecessary. Deleted.